packages/content/src/hub.ts

```diff
@@ -269,8 +297,13 @@ export function datasetToItem(dataset: DatasetResource): IItem {
     id: itemId,
     owner: owner as string,
     orgId,
-    created: (created || createdAt) as number,
-    modified: (modified || updatedAt) as number,
+    created: created as number,
+    // for feature layers, modified will usually come from the layer so
+    // we prefer itemModified, but fall back to modified if it came from the item
+    modified: (itemModified ||
```

Comment:
The nested ternary is hard to read, might be easier on the eyes written as

```js
modified: (itemModified || (modifiedProvenance === "item.modified" && modified) as number,
```

Reply:
I agree, and that's actually how I had it written originallyI.  think I when I changed it I was being preemptively defensive against the insults TS was going to lob at me for having the audacity to assign `false` to a `number`. Resolved in 02d08c69e336e9e960e7ecd3d441d9148996fb9a